pypeit/core/coadd.py

```diff
@@ -2446,13 +2494,38 @@ def ech_combspec(waves, fluxes, ivars, masks, weights_sens, nbest=None,
         None)
     """
 
-# TODO: Please leave this commented docstring entry here for now.
-#        merge_stack: bool, default=False,
-#            Compute an experimental combine of the high S/N combined orders in addition to the default algorithm,
-#            which is to compute one giant stack using all order overlaps
 
-    # output filenams for fits and QA plots
-    #outfile_order = outfile.replace('.fits', '_order.fits') if outfile is not None else None
+    # Notes on object/list formats
```

Comment:
I put the relevant bits in the docstring. These notes are mainly inline comments to understand all the list based data structures since that was quite complicated.